container/apptabs.go

```diff
@@ -22,6 +22,10 @@ var _ fyne.Widget = (*AppTabs)(nil)
 // Since: 1.4
 type AppTabs struct {
 	baseTabs
+	// Deprecated: 'Items' has been moved into an embedded type.
```

Comment:
I don't think this is wise. If you remove this then it will no longer be possible to do:

```go
&AppTabs{
    Items: ....
}
```

It's not just a case that the field will have moved, but that the shared type is private so it cannot be initialised externally.

Reply:
Yes it will no longer be possible to set `Items` in the struct initialization, devs will need to set it afterwards as the comment on line 27 details the release notes will also need to mention this.
It can be initialized externally as the baseTabs is embedded so `a.Items = ...` will work.
It does not seem unwise to me to trade off setting `Items` in the struct initialization for simpler and more maintainable code.
I don't see the issue with moving `Item` initialization onto a separate line (`a.Items = ...` or `a.SetItems(...)`) as struct initialization alone is not sufficient to initialize AppTabs anyway - the dev will still need to call `a.ExtendBaseWidget(a)`

Reply:
> the dev will still need to call a.ExtendBaseWidget(a)

This should not be required, the `CreateRenderer` or `MinSize` will take care of this like other widgets won't it?
That will only be needed if they want to extend further with their own code.

Reply:
Moot